build/features.py

```diff
@@ -825,42 +827,39 @@ def enabled(self, build):
 
     def add_options(self, build, vars):
         vars.Add('shoutcast', 'Set to 1 to enable live broadcasting support', 1)
-        vars.Add('shoutcast_internal', 'Set to 1 to use internal libshout', 0)
 
     def configure(self, build, conf):
         if not self.enabled(build):
             return
 
         build.env.Append(CPPDEFINES='__BROADCAST__')
 
-        build.flags['shoutcast_internal'] = util.get_flags(build.env, 'shoutcast_internal', 0)
-        if build.platform_is_linux and not int(build.flags['shoutcast_internal']):
-            # Check if system lib is lower 2.4.2 or 2.4.3 and not suffering bug
+        if build.platform_is_linux:
+            # Check if system lib is lower 2.4.2 or at least 2.4.4 and not suffering bug
             # https://bugs.launchpad.net/mixxx/+bug/1833225
-            if not conf.CheckForPKG('shout', '2.4.3'):
+            if not conf.CheckForPKG('shout', '2.4.4'):
                 if conf.CheckForPKG('shout', '2.4.2'):
```

Comment:
Thinking about it, do package maintainers rebuild all dependant packages if a library is updated? If not, maybe we should always use internal libshout if the version is below 2.4.4 even if 2.4.1 is currently installed?